src/backend/gencomm.py

```diff
@@ -11,8 +11,16 @@
 ## loop through the derived and basic types to generate individual
 ## pack functions
 derived_types = [ "hvector", "blkhindx", "hindexed", "contig", "resized" ]
-
-
+type_ops = {'_Bool': {'REPLACE', 'LAND', 'LOR', 'LXOR'},
+            'char': {'REPLACE'},
+            'wchar_t': {'REPLACE'},
+            'int8_t': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX', 'LAND', 'LOR', 'LXOR', 'BAND', 'BOR', 'BXOR'},
+            'int16_t': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX', 'LAND', 'LOR', 'LXOR', 'BAND', 'BOR', 'BXOR'},
+            'int32_t': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX', 'LAND', 'LOR', 'LXOR', 'BAND', 'BOR', 'BXOR'},
+            'int64_t': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX', 'LAND', 'LOR', 'LXOR', 'BAND', 'BOR', 'BXOR'},
+            'float': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX'},
+            'double': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX'},
+            'long double': {'REPLACE', 'SUM', 'PROD', 'MIN', 'MAX'}}
```

Comment:
Why is `byte` not listed here? Just curious since it is a basic datatype.

Reply:
Not all types are passed to the backend.  For example, you'll notice that plain `int` is missing too.  It's converted to fewer types by the frontend before being passed to the backend.

Reply:
Ok, yes. Thank you.